R/carehomes.R

```diff
@@ -75,6 +75,33 @@ NULL
 ##'   for a set period, provide two dates and two
 ##'   rates with the second rate equal to 0.
 ##'
+##' @param strain_rel_gamma_A Vector of relative rates of gamma_A for each
+##'   strain modelled. If `1` all strains have same rates. Otherwise vector of
+##'   same length as `strain_transmission`, with entries that determines the
+##'   relative scaling of the defaults for each strain.
+##'
+##' @param strain_rel_gamma_P Vector of relative rates of gamma_P for each
+##'   strain modelled. If `1` all strains have same rates. Otherwise vector of
+##'   same length as `strain_transmission`, with entries that determines the
+##'   relative scaling of the defaults for each strain.
+##'
+##' @param strain_rel_gamma_C_1 Vector of relative rates of gamma_C_1 for each
+##'   strain modelled. If `1` all strains have same rates. Otherwise vector of
+##'   same length as `strain_transmission`, with entries that determines the
+##'   relative scaling of the defaults for each strain.
+##'
+##' @param strain_rel_gamma_C_2 Vector of relative rates of gamma_C_2 for each
+##'   strain modelled. If `1` all strains have same rates. Otherwise vector of
+##'   same length as `strain_transmission`, with entries that determines the
+##'   relative scaling of the defaults for each strain.
+##'
+##' @param strain_rel_severity Vector of relative severity for each
```

Comment:
can we rename this strain_rel_p_D please (severity can be measured in many ways and here we are really only modifying the probability of death so will be clearer I think)

Reply:
actually seeing the rest of the code maybe this is fine but then in the documentation can you say it only is applied to probability of death please